src/main/java/com/atlassian/db/replica/api/DualConnection.java

```diff
@@ -375,25 +375,45 @@ public PreparedStatement prepareStatement(String sql, String[] columnNames) thro
     @Override
     public Clob createClob() throws SQLException {
         checkClosed();
-        throw new ReadReplicaUnsupportedOperationException();
+        if(compatibleWithPreviousVersion)
+            throw new ReadReplicaUnsupportedOperationException();
+        else
+            return connectionProvider
```

Comment:
```suggestion
        if (compatibleWithPreviousVersion) {
            throw new ReadReplicaUnsupportedOperationException();
        } else {
            return connectionProvider
```

Reply:
IMO it's noisier, but I will follow the convention

Reply:
It's `pythonier`.

Reply:
Maybe it's the same?
<img width="786" alt="image" src="https://user-images.githubusercontent.com/76737781/112134595-33e2f300-8bcd-11eb-8cb6-0ed28ace24cd.png">